components/espcoredump/src/core_dump_flash.c

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 #include <string.h>
+#include <assert.h>
+#include "esp_core_dump.h"
```

Comment:
Was these includes strictly required?  `_Static_assert` can be used without them